### DISCUSSION TITLE
Sound export

### DISCUSSION THREAD

**Discussion by ypapafi2022** [iimachines/Maya2glTF#183] [Q&A]
Hi
I would like to know if there is any way to export sound along with gltf animation from Maya.
thank you

**Comment by ziriax** [comment]
I need a bit more context here, because the GLTF spec doesn't contain anything related to sound.

The exporter does output an extra JSON file with high level information about the animation clips, so that could be easily used to associated sound with clips, if that is what you mean.

There is a discussion and open pull request from Microsoft:
https://github.com/KhronosGroup/glTF/issues/1582
https://github.com/KhronosGroup/glTF/pull/1400

Also, do you have an example Maya scene that contains sound?
